pkg/message/message_test.go

```diff
@@ -37,3 +38,29 @@ func TestChecksumReadCommand(t *testing.T) {
 	assert.Equal(t, expected, calcChecksum)
 	t.Log(len(input))
 }
+
+func TestMakeTorqueData(t *testing.T) {
+	type args struct {
+		message [24 * 4]byte
+	}
+	tests := []struct {
+		name string
+		args args
+		want *TorqueData
+	}{
+		{
+			name: "InstanceCheck",
+			want: &TorqueData{
+				header:   HeaderByte,
+				dataInfo: BulkParamSendMsg,
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := MakeTorqueData(tt.args.message); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("MakeTorqueData() = %v, want %+v", got, tt.want)
+			}
+		})
+	}
```

Comment:
nit: is a table driven test necessary here for testing a constructor function? It seems unlikely that many additional test cases would be added here, so it might be overkill.

Something much simpler could be

```suggestion
        testMessage :=[24 * 4]byte{}
	r := MakeTorqueData(testMessage)
	assert.Equals(t,HeaderByte, r.header)
	assert.Equals(t, BulkParamSendMsg,r.dataInfo)
	assert.Equals(t, testMessage,r.message)
```

Reply:
Hey! Thanks for reviewing the PR.
Well, Yes I'm learning Go and looking for some projects to contribute.
I liked the way you mentioned all the issues in your project, which was easy for me to understand what needs to be done and yes I liked the design and your addition to makefile and GoLint. 
Hence, decided to start contributing!

Also, made changes according to the comments!
Request/Suggestion: Can you please make the Readme more informative so the contributors can easily understand what you are trying to build?